### PR TITLE
Prepare 1.6.0

### PR DESCRIPTION
**Pull request by ricwo**
**Proposed changes**:
- bump version to 1.6.0

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa-sdk#code-style) for instructions)

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -7,11 +7,14 @@ This project adheres to `Semantic Versioning`_ starting with version 0.11.0.
 .. _master-release:
 
 
-[Unreleased 1.6.0a1] - `master`_
+[Unreleased 1.6.1a1] - `master`_
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. note:: This version is not yet released and is under active development.
 
+[1.6.0] - 2019-12-14
```
Comment (ricwo) [inline CHANGELOG.rst]:
```suggestion
[1.6.0] - 2019-12-16
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -7,11 +7,14 @@ This project adheres to `Semantic Versioning`_ starting with version 0.11.0.
 .. _master-release:
 
 
-[Unreleased 1.6.0a1] - `master`_
+[Unreleased 1.6.1a1] - `master`_
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. note:: This version is not yet released and is under active development.
 
+[1.6.0] - 2019-12-16
```
Comment (ricwo) [inline CHANGELOG.rst]:
```suggestion
[1.6.0] - 2019-12-17
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -7,11 +7,14 @@ This project adheres to `Semantic Versioning`_ starting with version 0.11.0.
 .. _master-release:
 
 
-[Unreleased 1.6.0a1] - `master`_
+[Unreleased 1.6.1a1] - `master`_
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. note:: This version is not yet released and is under active development.
 
+[1.6.0] - 2019-12-17
```
Comment (ricwo) [inline CHANGELOG.rst]:
```suggestion
[1.6.0] - 2019-12-18
```

**Comment by tmbo** [comment]
Happy to merge if it is ready to go

**Pull request merged**